symbolic-base/src/ZkFold/Symbolic/Data/FieldElement.hs

```diff
@@ -105,3 +105,12 @@ instance Symbolic c => BinaryExpansion (FieldElement c) where
   fromBinary bits =
     FieldElement $ symbolicF bits (Par1 . foldr (\x y -> x + y + y) zero)
       $ fmap Par1 . horner . fromVector
+
+createRangeConstraint :: Symbolic c => FieldElement c -> BaseField c -> FieldElement c
```

Comment:
This might be not the best place to define this function, I'd move it closer to its place of usage